BUILDING.md

```diff
@@ -8,6 +8,21 @@ You can skip to the [setup guide for Kubernetes](QUICKSTART-EKS.md) or the [setu
 
 ### Dependencies
 
+#### Adequate Disk Space
+
+The build process artifacts and resulting images can consume in excess of 4GB in the local directory.
+
+#### Operating System Packages
+
+The Rust tooling requires certain operating system packages to be installed in order to complete builds.  
+Error messages from the Rust tooling do not clearly identify the issues.
+
+Ensure the following OS packages are installed:
+
+- `build-essential`
```

Comment:
Fedora doesn't have a direct equivalent to this. It might be better to split it into separate commands, with an Ubuntu example and a Fedora example.